.github/workflows/smoke-test-package.yaml

```diff
@@ -0,0 +1,236 @@
+name: Smoke test package rpm/deb
+run-name: Smoke test package rpm/deb
+
+on:
+  workflow_dispatch:
+    inputs:
+      release-version:
+        description: |
+          Override the build version from what is in the Makefile.
+          Use this override to specify a tag for build debugging purposes.
+        type: string
+
+  # TODO(vapopov): delete before merge.
+  push:
+    branches:
+      - vapopov/smoke-test-package
+
+  merge_group:
+    paths:
+      - 'lib/autoupdate/**.go'
+      - 'tool/teleport-update/**.go'
+      - 'go.mod'
+      - 'go.sum'
+      - 'build.assets/Makefile'
+      - 'build.assets/Dockerfile*'
+      - 'Makefile'
+
+jobs:
+
+  build:
```

Comment:
Was this copied from another workflow? It'd be better to call another workflow from this one (reusable workflow) to ensure that the logic is consistent with release logic.

Reply:
I tried to reuse the existing workflows, but they are different and would require moving some parts from `teleport.e` to `teleport` to make them accessible for both repositories. As a result, I just created a new one - pretty simple:
 - `make release`
 - `make deb`
 - `make rpm-unsigned`
 - run the test script (add the production repo, install Teleport using apt, yum, or zypper, then upgrade/downgrade using the previously built package)